src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -16,24 +16,32 @@ public class Line extends BaseEntity {
     private String color;
     @Embedded
     private final Sections sections = new Sections();
+    private Integer overFare;
 
     public Line() {
     }
 
     public Line(String name, String color) {
         this.name = name;
         this.color = color;
+        this.overFare = 0;
     }
 
     public Line(String name, String color, Station upStation, Station downStation, int distance) {
+        this(name, color, upStation, downStation, distance, 0);
+    }
+
+    public Line(String name, String color, Station upStation, Station downStation, int distance, int overFare) {
         this.name = name;
         this.color = color;
+        this.overFare = overFare;
         sections.add(new Section(this, upStation, downStation, distance));
     }
 
     public void update(Line line) {
```

Comment:
이렇게 업데이트하면, overFare에는 항상 0이 들어가지 않을까요...? 추가로, service layer에서 Line 객체를 다시 만들어서 넣어주는 형태인데, 아래와 같이
```suggestion
    public void update(String name, String color) {
```
로 해주면 중간에 객체를 만들지 않아도 처리할 수 있지 않을까요...?

Reply:
말씀하신대로 `update()` 내부에서 `overFare`는 Update하지 않도록 수정하였습니다.
`update()`의 매개변수도 말씀하신대로 굳이 객체로 받을 필요가 없어 보이네요 😭 
수정하겠습니다! 🏃